indico/web/client/js/widgets/tz_selector.js

```diff
@@ -0,0 +1,35 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+// eslint-disable-next-line import/unambiguous
+window.addEventListener('DOMContentLoaded', function() {
```

Comment:
```suggestion
window.addEventListener('DOMContentLoaded', () => {
```

Reply:
Btw, I think arrow callbacks can be enforced via ESLint.

Reply:
I [thought so too](https://github.com/indico/eslint-config-indico/blob/909231c41a7a347b85925f720f2715a967b91696/configs/main.yaml#L162C3-L162C30)... `prefer-arrow-callback: warn`